<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
background-image: url(:/CardImages/fondoNaranja.png);
}
#instrution{
color: black;
}
#temporizador{
color: black;
}
#score_pt{
color:black;
}
#points_number{
color:black;
qproperty-alignment: AlignCenter;
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="instrution">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bienvenido Ingrese su nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="temporizador">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tiempo</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>661</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="card_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_7">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>280</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="card_12">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>280</y>
       <width>90</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="score_pt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">black</string>
    </property>
    <property name="text">
     <string>Puntuaci√≥n:</string>
    </property>
   </widget>
   <widget class="QLabel" name="points_number">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>550</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
